src/OrchardCore/OrchardCore.Apis.GraphQL.Client/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,9 @@
+namespace OrchardCore.Apis.GraphQL.Client
+{
+    internal static class StringExtensions
+    {
+        public static string ToGraphQLStringFormat(this string value) {
+            return char.ToLower(value[0]) + value.Substring(1);
```

Comment:
It's already there: src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/Extensions/StringExtensions.cs